make/photon/prepare/templates/core/env.jinja

```diff
@@ -36,6 +36,7 @@ CORE_URL={{core_url}}
 CORE_LOCAL_URL={{core_local_url}}
 JOBSERVICE_URL={{jobservice_url}}
 CLAIR_URL={{clair_url}}
+CLAIR_ADAPTER_URL={{clair_adapter_url}}
```

Comment:
I think the CLAIR_DB_xxx are all useless?

Reply:
The same to `CLAIR_URL `

Reply:
clair adapter need the url of the clair and harbor core need the url of the clair adapter